UndertaleModTool/Editors/UndertaleRoomEditor/UndertaleRoomEditor.xaml.cs

```diff
@@ -1068,12 +1051,12 @@ private void SelectObject(UndertaleObject obj, bool focus = true)
                         break;
 
                     case GameObject gameObj:
-                        if (room.Flags.HasFlag(RoomEntryFlags.IsGMS2) || room.Flags.HasFlag(RoomEntryFlags.IsGM2024_13))
+                        if ((room.Flags.HasFlag(RoomEntryFlags.IsGMS2) || room.Flags.HasFlag(RoomEntryFlags.IsGM2024_13)) && !isInAllGameObjectsList)
                         {
                             resLayer = room.Layers.AsParallel()
-                                                  .WithExecutionMode(ParallelExecutionMode.ForceParallelism)
-                                                  .FirstOrDefault(l => l.LayerType is LayerType.Instances
-                                                      && (l.InstancesData.Instances?.Any(x => x.InstanceID == gameObj.InstanceID) ?? false));
+                                                    .WithExecutionMode(ParallelExecutionMode.ForceParallelism)
```

Comment:
This isn't really a review for this PR in particular, but does anyone know why there's parallelism here? Surely the overhead is usually higher than doing this non-parallel?